packages/backend/src/services/FeatureFlagService/FeatureFlags.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * Before using a FeatureFlag, include it as a member in this enum,
+ * to ensure you can use it with FeatureFlagService. Consider adding
+ * a short description of the feature flag and how it will be used.
+ *
+ * If the feature flag is no longer in use, remove it from this enum.
+ */
+export enum FeatureFlags {
```

Comment:
Should we expose this enum in common so it can be used in the front end? We'd need to build some equivalent context or something equivalent to this service on the FE to make good use of it. 

We sometimes have flags that are used in both places. If you think it's better to keep the BE and FE enums separate for any reason, I can get on board with that too. But if one of the goals is to keep track of which of these exist better, sharing this enum as a source of truth for that could be a good idea.

Reply:
Definitely! I left frontend as an exercise for the future since I'm not yet familiar with house we're using feature flags there, but I fully agree on principle, and there's no good reason to not include the enum in `common` from the start, even if there's some lag on actually using it in frontend - I'll make that change!

Reply:
Perfect, yea we dont need to use it there for now, but making the enum shared would be good. 👍